R/summary.pop_data.R

```diff
@@ -22,13 +23,14 @@ summary.pop_data = function(object, ...)
 
   ages =
     object %>%
-    distinct(.data$id, .data$age)
+    distinct(.data$id,.data$age)
```

Comment:
why remove the whitespace? Please follow the style guide: https://style.tidyverse.org/syntax.html#commas

Reply:
An oversight.